agent/command/s3_put.go

```diff
@@ -338,6 +353,18 @@ retryLoop:
 				}
 
 				fpath = filepath.Join(filepath.Join(s3pc.workDir, s3pc.LocalFilesIncludeFilterPrefix), fpath)
+
+				if s3pc.noOverwriteBool {
+					// check if the file exists
```

Comment:
I don't think this comment is necessary since it's clear from the code itself that it's checking for the existence of the remote file.